integration_tests/src/main/python/json_test.py

```diff
@@ -1454,3 +1454,11 @@ def test_spark_from_json_invalid_json():
     assert_gpu_and_cpu_are_equal_collect(
         lambda spark : spark.createDataFrame(data, 'json STRING').select(f.col('json'), f.from_json(f.col('json'), schema)),
         conf =_enable_all_types_conf)
+
+@allow_non_gpu(*non_utc_allow)
+def test_from_json_input_wrapped_in_whitespaces():
+    json_string_gen = StringGen(r'[ \r\n\t]{0,5}({"key": "[A-z]{0,5}"}|null|invalid|)[ \r\n\t]{0,5}')
```

Comment:
Will generate text that is either:
 * [A-z]{0,5}
 * "null"
 * "invalid"
 * Empty string

And each of these strings are surrounded by whitespace chars `[ \r\n\t]{0,5}`.

Reply:
Can be tested at https://onlinestringtools.com/generate-string-from-regex